deploy/common/deploy.sh

```diff
@@ -69,6 +69,7 @@ function generate_ca {
 #   - API server endpoint to ping
 function wait_for_central {
     LOCAL_API_ENDPOINT="$1"
+    local namespace=${2:-stackrox}
```

Comment:
Nit: This could be renamed to `central_namespace`

Reply:
I have seen that (both occurrences), but I figured the functions are short enough to not bloat the PR further. But if that is desired, I can for sure add the change in a follow-up PR. Would that be ok?

Reply:
Fine for me!